vividus-engine/src/test/java/org/vividus/BatchedPathFinderTests.java

```diff
@@ -54,8 +50,9 @@ void testFindPaths() throws IOException
         when(batchStorage.getBatchResourceConfigurations()).thenReturn(Map.of(batchKey, batchResourceConfiguration));
         List<String> testPaths = List.of("testPath");
         when(pathFinder.findPaths(batchResourceConfiguration)).thenReturn(testPaths);
-        Map<String, List<String>> actual = batchedPathFinder.findPaths();
         Map<String, List<String>> expected = Map.of(batchKey, testPaths);
-        assertEquals(expected, actual);
+        assertEquals(expected, batchedPathFinder.getPaths());
```

Comment:
how caching is tested? Shouldn't you verify invocation quantity?

Reply:
please check verifyNoMoreInteractions below